src/components/actions/filters/ItemFilterActions.ts

```diff
@@ -0,0 +1,37 @@
+import { IReduxAction } from '../../../utils/ReduxUtils';
+
+export const ItemFilterActions = {
+  add: 'ADD_ITEM_FILTER',
+  filter: 'FILTER_ITEMS',
+  remove: 'REMOVE_ITEM_FILTER'
+};
+
+export interface IItemFilterActionPayload {
```

Comment:
Question: it's possible to have a base action payload for this kind of payload with only an id. I think it's not the only one with only an id ?